trade-finance-manager-api/src/v1/api.js

```diff
@@ -1203,6 +1203,27 @@ const getUtilisationReportsReconciliationSummary = async (submissionMonth) => {
   return response.data;
 };
 
+/**
+ * Sends a payload to DTFS central API to update
+ * the status of one or more utilisation reports
+ * @param {import('../types/utilisation-report-service').ReportWithStatus[]} reportsWithStatus
+ * @param {import('../types/tfm-session-user').TfmSessionUser} user - The current user in stored in the session
+ * @returns {Promise<{ status: number }>}
+ */
+const updateUtilisationReportStatus = async (reportsWithStatus, user) => {
+  const { status } = await axios({
+    method: 'put',
+    url: `${DTFS_CENTRAL_API_URL}/v1/tfm/utilisation-reports/set-status`,
+    headers: headers.central,
+    data: {
+      user,
+      reportsWithStatus,
+    },
+  });
+
+  return { status };
```

Comment:
Not sure if there is a point to returning this - the status will always be `200` if this return statement is reached, so it feels slightly like a redundant return